<launch>
  <node name="kinect" pkg="pwcnet_ros" type="sample_node" required="true">
    <remap from="image" to="/kinect2/qhd/image_color"/>
    <param name="image_transport" value="raw"/>
  </node>
</launch>
